go/worker/storage/committee/node.go

```diff
@@ -540,16 +540,20 @@ func (n *Node) worker() { // nolint: gocyclo
 	defer close(n.diffCh)
 
 	// Wait for the common node to be initialized.
+	n.logger.Info("matevz0")
```

Comment:
Feel free to remove these debug statements :-)

Reply:
Oops :wink: